src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2239,7 +2240,7 @@ void processContextElement
       {
         ContextAttribute* ca = new ContextAttribute(aP);
 
-        buildGeneralErrorResponse(ceP, ca, responseP, SccInvalidParameter,
+        buildGeneralErrorResponse(ceP, ca, responseP, SccInvalidModification,
```

Comment:
Interesting point to think about it... :)

This change introduces a slight backward compability break, as we are returning 422 in places where 472 was returned previously. However, in this case we are moving from a general error (invalid parameter, in general) to a more specific error (invalid modification), so I think is actually an improvement.

We could avoid this and preserve previous NGSIv1 behaviour using the "if (apiVersion == v2)" technique already used in other PRs. However, in this particular case I think we should leave the modification as @rbl219 suggest.

Opinions?